website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -38,25 +38,33 @@ This guides makes the following assumptions:
 
 - You are familiar with the [General Upgrade Process](/docs/upgrading/instructions/general-process).
 - You have the ability to run Consul CLI commands. 
+- If ACLs are in use then you possess a token with at least `operator:read` permissions.
 
 ## Considerations
 
 The main breaking changes in Consul Enterprise 1.10 that needs special handling during upgrading are
 the licensing changes outlined on the [Specific Version Details](/docs/upgrading/upgrade-specific#consul-1-10-0)
 page. There are notes on that paged about other changes that might cause issues during an upgrade as well.
-You can find more granular details in the full [changelog](https://github.com/hashicorp/consul/blob/release/release/1.10.0/CHANGELOG.md#1100-june-22-2021).
+You can find more granular details in [FAQ](/docs/enterprise/license/faq) as well as the full 
+[changelog](https://github.com/hashicorp/consul/blob/release/release/1.10.0/CHANGELOG.md#1100-june-22-2021).
 Looking through these changes prior to upgrading is highly recommended.
 
 ## Procedures
 
+~> When any of the steps below call for upgrading components, we assume you follow
+the normal safe upgrade procedure you normally use for upgrades. The basic server
+upgrade procedure is documented [here](/docs/upgrading/instructions/general-process).
```

Comment:
Nit, instead of linking to the word "here" suggest rewording the sentence to "Refer to our documentation for [basic server upgrade process](), and for [autopilot assisted upgrades]()"

Reply:
Done.